data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -159,19 +163,25 @@ def _create_score_data(
         states_df: pd.DataFrame,
         score_df: pd.DataFrame,
     ) -> pd.DataFrame:
-        # merge state with counties
-        logger.info("Merging state with county info")
-        county_state_merged = counties_df.merge(
-            states_df, on="State Abbreviation", how="left"
-        )
 
-        # merge state + county with score
-        score_county_state_merged = score_df.merge(
-            county_state_merged,
+        logger.info("Merging county info with score info")
+        score_county_merged = score_df.merge(
+            # We drop state abbreviation so we don't get it twice
+            counties_df[["GEOID", "County Name"]],
             on="GEOID",  # GEOID is the county ID
             how="left",
         )
 
+        logger.info("Merging state info with county-score info")
+        # Here, we need to join on a separate key, since there's no
+        # GEOID for the territories.
+        score_county_merged["State Code"] = score_county_merged[
+            self.GEOID_TRACT_FIELD_NAME
+        ].str[:2]
+        score_county_state_merged = score_county_merged.merge(
+            states_df, left_on="State Code", right_on="State Code", how="left"
```

Comment:
nitty nit: you know i get nervous whenever i see string field names floating around. maybe make these module-level constants (i.e., at the top of the file) or class-level constants (i.e., at the start of the class)? Probably class-level makes a little more sense.

Reply:
`For future reference, we could also refactor this code so that the FIPS / State or Territory / County info gets created as an ETL process and joined in etl_score, rather than added in post like this. That would be a bit more consistent and automatically parallelized.` -> this makes a toooon of sense to me. Wondering if we can do this in The Big Refactor?

Reply:
Totally agree! And after we have tests would be good.